buildSrc/src/main/kotlin/Deps.kt

```diff
@@ -42,7 +42,7 @@ object Deps {
             const val material = "com.google.android.material:material:$materialDesignVersion"
             const val lifecycle = "androidx.lifecycle:lifecycle-extensions:$androidLifecycleVersion"
             const val exifInterface = "androidx.exifinterface:exifinterface:$androidExifInterface"
-            const val mediaFilePicker = "com.nbsp:materialfilepicker:$androidMediaFilePicker"
+            const val mediaFilePicker = "com.github.Lobynya:MaterialFilePicker:$androidMediaFilePicker"
```

Comment:
please change usage to https://github.com/icerockdev/MaterialFilePicker - organization's fork more stable variant :)

Reply:
fixed